src/app/youtube/components/sort/sort.component.ts

```diff
@@ -18,32 +18,31 @@ export class SortComponent implements OnInit, OnDestroy {
     return this.sortService.classes
   }
 
-  private sortVisibilitySubscr!: Subscription
+  private sortVisibilitySubscription!: Subscription
 
   constructor(private sortService: SortbarManagerService) {}
 
   ngOnInit(): void {
-    this.sortVisibilitySubscr = this.sortService.sortVisibility$.subscribe(
-      isVisible => {
+    this.sortVisibilitySubscription =
+      this.sortService.sortVisibility$.subscribe(isVisible => {
         this.isVisible = isVisible
-      }
-    )
+      })
```

Comment:
take a look at `async` pipe. It simplifies this logic a lot.

Reply:
try to use it in your template

Reply:
yes, async pipe do a lot, fixed